test-requirements.txt

```diff
@@ -4,4 +4,5 @@ pytest-env >= 0.6.0
 pytest-xdist >= 2.0
 flaky >= 3.0
 responses >= 0.14.0
-PyJWT >= 2.3.0
\ No newline at end of file
+PyJWT >= 2.3.0
+py2neo >= 2021.2.3
```

Comment:
This can be left out of the test requirements. The pipeline installs all extras before testing tasks in the task library.
```suggestion
```

Reply:
Without `py2neo` in `test-requirements.txt` I get
```
ImportError while importing test module '/root/project/tests/tasks/neo4j/test_neo4j_tasks.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.7/site-packages/prefect/tasks/neo4j/__init__.py:6: in <module>
    from prefect.tasks.neo4j.neo4j_tasks import Neo4jRunCypherQueryTask
/usr/local/lib/python3.7/site-packages/prefect/tasks/neo4j/neo4j_tasks.py:3: in <module>
    from py2neo import Graph
E   ModuleNotFoundError: No module named 'py2neo'
```

Reply:
You can add the following in `tests/tasks/neo4j/__init__.py` to only run these tests during the task library test stage:
```
import pytest

pytest.importorskip("py2neo")
```